ddtrace/contrib/django/middleware.py

```diff
@@ -18,6 +18,27 @@
 log = logging.getLogger(__name__)
 
 
+class TraceExceptionMiddleware(object):
```

Comment:
Ok, i'll do it! But actually why are we doing this, is this to support all versions of django?